### PR TITLE
Split webui build into bundling and building to support arm64 builds.

### PR DESCRIPTION
**Pull request by haiqi96**
# Description
Split the webui build process into two stages webui-bundle and webui to support building a package on arm64 architecture. The bundling stage requires meteor which is only available x64 architecture, so the strategy is to bundle the package on x86_64 and build from arm64 machine

# Validation performed
Built AMI with the change and verified that CLP can compress and search results is displayed on webui

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -157,8 +157,8 @@ tasks:
   nodejs:
     vars:
       NODEJS_VERSION: "14.21.3"
-      TAR_FILE_NAME: "node-v14.21.3-linux-x64.tar.xz"
-      TAR_FILE_PATH: "{{.NODEJS_BUILD_DIR}}/node-v14.21.3-linux-x64.tar.xz"
+      NODEJS_ARCH: "{{ if eq ARCH \"arm64\" }}arm64{{ else }}x64{{ end }}"
```
Comment (kirkrodrigues) [inline Taskfile.yml:160]: Where is `ARCH` defined?

File: Taskfile.yml [hunk 2]
```diff
@@ -168,15 +168,26 @@ tasks:
...
+        mv "{{.NODEJS_BUILD_DIR}}/node-v{{.NODEJS_VERSION}}-linux-{{.NODEJS_ARCH}}"
+        "{{.NODEJS_BUILD_DIR}}/node"
       - "rm -f '{{.NODEJS_BUILD_DIR}}/{{.TAR_FILE_NAME}}'"
     sources:
       - "{{.NODEJS_BUILD_DIR}}/node/**/*"
 
   webui:
     deps:
+      - "webui-bundle"
       - "nodejs"
```
Comment (kirkrodrigues) [inline Taskfile.yml:180]:
```suggestion
      - "nodejs"
      - "webui-bundle"
```

File: Taskfile.yml [hunk 3]
```diff
@@ -168,15 +168,26 @@ tasks:
...
     dir: "components/webui"
+    cmds:
+      - "rm -rf '{{.WEBUI_BUILD_DIR}}/bundle/'"
```
Comment (kirkrodrigues) [inline Taskfile.yml:183]: Shouldn't this line be part of the `webui-bundle`?

File: Taskfile.yml [hunk 4]
```diff
@@ -168,15 +168,26 @@ tasks:
...
+        "{{.NODEJS_BUILD_DIR}}/node"
       - "rm -f '{{.NODEJS_BUILD_DIR}}/{{.TAR_FILE_NAME}}'"
     sources:
       - "{{.NODEJS_BUILD_DIR}}/node/**/*"
 
   webui:
     deps:
+      - "webui-bundle"
       - "nodejs"
     dir: "components/webui"
```
Comment (kirkrodrigues) [inline Taskfile.yml:181]:
```suggestion
```

**Review by kirkrodrigues — CHANGES_REQUESTED** [review]

File: Taskfile.yml (hunk 1 above)
Comment (haiqi96) [reply Taskfile.yml:160]: ARCH is a actually a function provided by TaskFile, see https://taskfile.dev/usage/#platform-specific-tasks-and-commands

**Review by kirkrodrigues — APPROVED** [review]
Commit msg: `Split webui build into bundling and building to support arm64 builds. (#276)`?

**Pull request merged**